modules/samples/connectedcup/component/agent/src/main/webapp/index.jsp

```diff
@@ -1,140 +1,134 @@
-<html>
-    <head>
-
-        <link rel="stylesheet" href="bootstrap.min.css">
-        <link rel="stylesheet" href="jquery-ui.css">
-        <script src="jquery-1.12.0.min.js"></script>
-        <script src="jquery-ui.js"></script>
-
-        <style>
-
-            .filler {
-                height:200px;
-                background-color:black;
-            }
-        </style>
-
-    </head>
-    <body>
-    <%
-        String deviceId = request.getParameter("deviceId");
-        if(deviceId != null)request.getSession().setAttribute("deviceId",deviceId);
-
-        String deviceOwner = request.getParameter("deviceOwner");
-        if(deviceOwner != null)request.getSession().setAttribute("deviceOwner",deviceOwner);
-
-        String token = request.getParameter("token");
-        if(token != null)request.getSession().setAttribute("token",token);
+<%--
+  ~ Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~
+  ~ WSO2 Inc. licenses this file to you under the Apache License,
+  ~ Version 2.0 (the "License"); you may not use this file except
+  ~ in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied. See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  --%>
 
-    %>
-    <div class="row col-sm-offset-5">
-        <div class="col-md-4" ><h3>Connected Cup</h3></div>
+<html>
+<head>
+    <title>Connected Coffee Cup</title>
+
+    <link rel="stylesheet" href="css/coffee.css">
+
+    <!--[if lt IE 9]>
+    <script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script>
+    <![endif]-->
+</head>
+
+<body>
+<%
+    String deviceId = request.getParameter("deviceId");
+    if (deviceId != null) {
+        request.getSession().setAttribute("deviceId", deviceId);
+    }
+
+    String deviceOwner = request.getParameter("deviceOwner");
+    if (deviceOwner != null) {
+        request.getSession().setAttribute("deviceOwner", deviceOwner);
+    }
+
+    String token = request.getParameter("token");
+    if (token != null) {
```

Comment:
Use the notation:
if (token) {} -> this will check for null, "", undefined and false